spec/tweetstream/client_authentication_spec.rb

```diff
@@ -2,17 +2,19 @@
 
 describe TweetStream::Client do
   before do
-    @stream = double('EM::Twitter::Client',
-                     :connect => true,
-                     :unbind => true,
-                     :each => true,
-                     :on_error => true,
-                     :on_max_reconnects => true,
-                     :on_reconnect => true,
-                     :connection_completed => true,
-                     :on_no_data_received => true,
-                     :on_unauthorized => true,
-                     :on_enhance_your_calm => true,
+    @stream = double(
+      'EM::Twitter::Client',
+      :connect => true,
+      :unbind => true,
+      :each => true,
+      :on_error => true,
+      :on_max_reconnects => true,
+      :on_reconnect => true,
+      :connection_completed => true,
+      :on_no_data_received => true,
+      :on_unauthorized => true,
+      :on_enhance_your_calm => true,
+      :on_close => true,
```

Comment:
RuboCop forced me to do this. I got RuboCop offenses for each @stream = double because RuboCop wanted the ending ")" to be in same column as starting "(". It wouldn't accept my code. After I changed all that, @sferik  changed the master branch to have all the indents go the way they are now.